### PR TITLE
Added is spoken things AB#2288

### PR DESCRIPTION
**Pull request by RubenGeo**

### CODE REVIEW

File: interfaces/AW-App/src/app/validation-components/upload-data/upload-data.component.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
   >
     {{ 'validation.upload-data.upload-result'|translate }}
   </dialogue-turn>
-  <dialogue-turn *ngIf="!uploadDataStored" [isSpoken]="noUploadDataStored">
+  <dialogue-turn *ngIf="!uploadDataStored" [isSpoken]="!noUploadDataStored">
```
Comment (elwinschmitz) [inline interfaces/AW-App/src/app/validation-components/upload-data/upload-data.component.html:19]: The `isSpoken`-condition needs to be the same as the `*ngIf` one..

File: interfaces/AW-App/src/app/validation-components/validate-fsp/validate-fsp.component.html [hunk 2]
```diff
@@ -37,11 +37,18 @@
     {{ 'validation.validate-fsp.no-fsp'|translate }}
     </p>
   </dialogue-turn>
+<<<<<<< HEAD
```
Comment (elwinschmitz) [inline interfaces/AW-App/src/app/validation-components/validate-fsp/validate-fsp.component.html:40]: Some merge-resolve-quirk here. :/

Comment (RubenGeo) [reply interfaces/AW-App/src/app/validation-components/validate-fsp/validate-fsp.component.html]: Ahh, missing  a ctrl + s

File: interfaces/AW-App/src/app/validation-components/upload-data/upload-data.component.html (hunk 1 above)
Comment (elwinschmitz) [reply interfaces/AW-App/src/app/validation-components/upload-data/upload-data.component.html:19]: @RubenGeo

Comment (RubenGeo) [reply interfaces/AW-App/src/app/validation-components/upload-data/upload-data.component.html:19]: I read this line like 3 times thinking the conditions are already the same

**Pull request merged**